packages/itwinui-react/src/core/Stepper/Stepper.tsx

```diff
@@ -44,6 +45,38 @@ export type StepperProps = {
    *  Click handler on completed step.
    */
   onStepClick?: (clickedIndex: number) => void;
+  /**
+   * Callback that can provide additional props for `<li>` representing a step.
+   */
+  stepProps?: (index: number) => React.ComponentProps<'li'>;
+  /**
+   * Allows props to be passed for iui-stepper-step-track-content
```

Comment:
Remove these class names when describing the props. You can just call this the track content wrapper or circle wrapper or something. The other new props in this file also need this change

Reply:
done